src/styles/blocks/card.scss

```diff
@@ -0,0 +1,97 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: $card-width;
+  padding: 32px 16px 16px;
+  margin-bottom: 46px;
+  margin-right: 48px;
```

Comment:
Card it's a BEM block, BEM block can't have margins!!!

Reply:
rude mistake fixed and updated!